shared-bindings/spitarget/SPITarget.c

```diff
@@ -0,0 +1,182 @@
+#include "shared-bindings/microcontroller/Pin.h"
+#include "shared-bindings/spitarget/SPITarget.h"
+#include "shared-bindings/time/__init__.h"
+#include "shared-bindings/util.h"
+
+#include "shared/runtime/buffer_helper.h"
+#include "shared/runtime/context_manager_helpers.h"
+#include "shared/runtime/interrupt_char.h"
+
+#include "py/mperrno.h"
+#include "py/mphal.h"
+#include "py/obj.h"
+#include "py/objproperty.h"
+#include "py/runtime.h"
+
+//| class SPITarget:
+//|     """Serial Peripheral Interface protocol target"""
```

Comment:
Please include a basic example here so that we can see what the API use looks like.

Reply:
I added an example in the `spitarget/__init__.c` file since that's the analog for where `i2ctarget`'s example is placed, let me know if the example code looks alright.